docs/conf.py

```diff
@@ -16,8 +17,31 @@
     "sphinx_comments",
     "sphinx_copybutton",
 ]
+html_logo = (
+    "https://raw.githubusercontent.com/ComPWA/ComPWA/04e5199/doc/images/logo.svg"
```

Comment:
The logo overwrites the title in the top of the navigation bar. To avoid that, see
https://github.com/ComPWA/ampform/blob/1a7d34fcbcf638f5d4b2b94fb0d65badde347900/docs/conf.py#L249